Neos.TimeableNodeVisibility/Resources/Private/Translations/en/NodeTypes/Timeable.xlf

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+	<file original="" product-name="Neos.TimeableNodeVisibility" source-language="en" datatype="plaintext">
+		<body>
+			<trans-unit id="properties.enableAfterDateTime" xml:space="preserve">
+				<source>Show after</source>
+			</trans-unit>
+			<trans-unit id="properties.disableAfterDateTime" xml:space="preserve">
+				<source>Hide after</source>
```

Comment:
Is it intentional that we still use the `Hide` as label name like before even though we understand it to be disable?

Reply:
What's your suggestion? "Disable after"?